### PR TITLE
feat(resume): материализация пула резюме на hh.ru + связка кластер-резюме

### LINKED ISSUE #754: feat(resume): материализация пула резюме на hh.ru + связка кластер-резюме
Часть эпика #750 (адаптивные резюме: пул под кластеры).
**Зависит от A3** (генерация содержимого).

## Задача

Материализовать пул: создать на hh.ru резюме по одному на кластер, с содержимым из A3,
и связать каждое резюме с его кластером в конфиге.

## Размер пула зафиксирован: 4 резюме (#752, замер 2026-08-29)

Research по живой выдаче hh.ru (432 уникальные вакансии) дал **4 кластера, а не 3**:

| # | Кластер | Эксклюзивных вакансий | Медиана ЗП (RUB) |
|---|---|---|---|
| 1 | AI/LLM-инженер | 66 | 200 000 |
| 2 | Data Science / ML | 61 | 337 500 |
| 3 | Data-инженер | 83 | 270 000 |
| 4 | Python-бэкенд | 55 | 127 500 |

Data Science/ML — самостоятельный кластер, не часть AI/LLM (61 вакансия матчится только
туда). DevOps/Infra проверен и отвергнут — растворяется внутри AI/Data-вакансий.
Полный отчёт — в комментарии к #752.

4 резюме против лимита hh.ru ~20 — запаса достаточно, но **явный предел на прогон
всё равно обязателен** (см. ниже: троттлинга у create/copy нет вовсе).

## Ключевое ограничение — лимит резюме hh.ru

Лимит ~20 задокументирован в `src/hhru_bot/copy_resume.py:5-7` и `:749`:
кнопка «Дублировать» НЕ рендерится при достижении лимита — **это единственный видимый
признак**, её отсутствие трактуется как отказ, а не как «селектор устарел».

**Программной проверки счётчика резюме не существует.** Команда обязана останавливаться
по этому косвенному признаку и НЕ пытаться угадать счётчик.

## Ключевое ограничение — троттлинга нет

Дневных лимитов на `copy-resume`/`create-resume`/`edit-*` нет вовсе
(`commands/copy_resume.py:6-7`, согласовано в cli-spec §3.3). Единственный тормоз —
`--force`/TTY-подтверждение через `confirm_write` (`commands/copy_resume.py:63`).

Значит: команда создания пула **обязана иметь собственный явный предел** на число
создаваемых за прогон резюме и обязательный `--dry-run`. Не полагаться на throttle —
его здесь нет.

## Что переиспользовать

- `commands/copy_resume.py:30` — флаги `--resume --dry-run --force --write-config --slug --title`.
  `--write-config` дописывает секцию в `config.yaml` только после подтверждённого успеха.
- `commands/create_resume.py:13` — `--area --title`, `--write-config` у него ОТСУТСТВУЕТ
  (печатает сниппет, `create_resume.py:91`).
- `copy_resume.py:693-699` `copy_resume_on_hh(...)`, `create_resume.py:150-157` `create_resume_on_hh(...)`.

## Durable-гарантии — соблюдать, не обходить

- `history.start_command_run` lease с owner PID (#475): конкурентная durable-команда
  получает `CommandRunBusy`.
- `DurableMutationAttempt` / `before_click` seam (#476): `uncertain`-маркер резервируется
  только начиная с реального клика; pre-click ошибки остаются `failed` и не блокируют повтор.
- `has_unresolved_uncertain` блокирует повтор мутации. Для `create-resume` ключ —
  фиксированный `"account"` (`commands/create_resume.py:52-60`), потому что до присвоения
  hh.ru своего `resume_id` нет. Слепой повтор после `uncertain` создал бы ВТОРОЕ резюме
  (комментарий `:47-51`, #464 cycle-review).

Массовое создание пула проходит через эти же гарантии для каждого резюме. Не изобретай
обходной путь ради batch-режима.

## Связка кластер → резюме

Каждое резюме конфига должно знать свой кластер. Это разблокирует роутер: сейчас
`route_vacancies` (`apply/router.py:85`) включается только при `len(resumes) > 1`
(`commands/apply.py:77`) и выбирает по скорингу, не зная про кластеры.

## Ограничения

- **Файлы не удалять** — запрет пользователя на весь флот. Распространяется и на резюме:
  не закладывать сценарии, требующие `delete-resume` (у него известная необратимая
  блокировка #480).
- Обязательный `--dry-run`, показывающий полный план: сколько резюме, под какие кластеры,
  с какими заголовками.
- **WRITE на создание резюме РАЗРЕШЁН** пользователем явно (2026-08-29): воркер может
  реально создать резюме пула на живом hh.ru, отдельно переспрашивать не нужно.
  Разрешение узкое — оно покрывает **создание** и ничего больше. Любая другая мутация
  аккаунта (удаление, снятие с публикации, правка чужого резюме, отклики) под него
  НЕ попадает и требует отдельного разрешения в моменте.
- Перед боевым прогоном обязательно сначала `--dry-run` и сверка плана: сколько резюме,
  под какие кластеры, с какими заголовками. Создавать вслепую нельзя.
- Роутер требует подтверждённой identity резюме: `id`, `resume_id`, URL с префиксом
  `https://hh.ru/resume/`, не placeholder (`apply/router.py:60-68`). Созданные резюме
  обязаны этому удовлетворять, иначе выпадут из роутинга молча.
- Резюме в статусе `not_finished` исключаются из роутинга (`commands/apply.py:86-95`) —
  учесть, что свежесозданное резюме может быть черновиком.
- PR ≤ 1000 строк.

## Definition of Done

- `--dry-run` печатает полный план создания пула.
- Достижение лимита hh.ru обрабатывается как явный отказ с понятным сообщением.
- Явный предел на число создаваемых за прогон резюме реализован.
- Каждое резюме пула связано со своим кластером в конфиге.
- `ruff check` + `ruff format --check`, полный `pytest`, README перегенерён, зелёный CI.

## Оценка

- **LOC:** ~600–900 (src ~300–450 / tests ~300–450)
- **Размер:** XL
- **Время (код):** ~100–180 мин
- **Время (review):** ~90–120 мин — **база XL 45–60 ×2**
- **Время (итого):** ~190–300 мин

**Множитель ×2 к review обоснован** (Шаг 3.1 estimate.md): задача touch
**concurrency / process lock** — durable `command_runs` lease с owner PID (#475),
`DurableMutationAttempt`/`before_click` seam (#476), `has_unresolved_uncertain`.
Это ровно тот класс, где Codex гоняет edge cases до clean; исторический прецедент —
`#472 feat(commands): journal resume edit runs` — 1322 (src 443 / **tests 879**),
где тесты на durable-журнал заняли вдвое больше кода.

**Калибровка** (аналоги):
`#531 feat: optionally persist copied resume config` — 603 (src 229 / tests 372);
`#558 feat(copy-resume): support setting copied resume title` — 459;
`#470 feat(commands): journal resume mutations` — 576;
`#472 feat(commands): journal resume edit runs` — 1322.

**Риск превышения лимита 1000 строк — высокий**, из-за объёма durable-тестов
(прецедент #472). Если превышает: PR-1 — материализация пула + лимит/`--dry-run`;
PR-2 — связка кластер↔резюме в конфиге и проброс в роутер. Излишек — follow-up issue.

### PR DESCRIPTION
**Pull request by axisrow**
Реализует #754 (эпик #750, материализация пула резюме под кластеры).

## Что сделано

Новая команда `resume-pool` копирует одно уже заполненное базовое резюме
(`--source`) по разу на каждый недостающий кластер вакансий
(`resume_clusters.py`, #752/#753) — **через `copy-resume`** ("Дублировать"),
**не через `create-resume`**: тот создаёт пустой черновик без единой записи
опыта, непригодный для дальнейшей адаптации содержимого под кластер (issue
#769). См. [комментарий-корректировку к исходному телу issue #754](https://github.com/axisrow/hhru/issues/754#issuecomment-5460820962) —
текст issue ссылался на `create_resume.py` как на равноценную альтернативу,
это оказалось неверно после проверки применения #769.

- `resume_pool.py` — чистая `build_pool_plan()`: какие кластеры уже покрыты
  (по полю `resume.cluster` в конфиге), какие недостают, обрезка плана по
  `--limit`. Число кластеров **не хардкодится** нигде — код итерирует
  `resume_clusters.CLUSTERS` и использует `len(...)` динамически.
- `commands/resume_pool.py` — CLI: `--source --dry-run --force --limit
  --write-config`. Batch строго последователен и **останавливается на
  первом отказе/uncertain**: `has_unresolved_uncertain(source.resume_id,
  "copy_resume")` ключуется по resume_id ИСХОДНОГО резюме — общий для всех
  копий одного источника, поэтому durable-гарантии не обходятся ради
  batch-режима (issue п.7), guard проверяется заново перед каждой копией.
- `config_sections/cluster.py` + `ResumeConfig.cluster` — новая опциональная
  resume-секция, связывает резюме с кластером; валидируется на загрузке
  конфига через `resume_clusters.cluster_by_key`.
- `--write-config` дописывает `config.yaml` для каждой успешно созданной
  копии (id/resume_url/`cluster: "<ключ>"`); без флага — печатает YAML-
  фрагмент на каждую копию (как `copy-resume`/`create-resume`).
- Лимит резюме hh.ru (~20) обрабатывается косвенно, тем же механизмом, что
  и `copy-resume`: отсутствие кнопки «Дублировать» → `duplicate_action_missing:`
  reason, обычный pre-click `[FAIL]` без угадывания счётчика.

## Ничего не удаляется

`delete-resume` не вызывается нигде в реализации, локальные файлы не
удаляются ни при каком исходе (в т.ч. неудачном/uncertain).

## Проверка

- `ruff check` + `ruff format --check` (src tests scripts) — зелёные.
- Полный `pytest -n 4 --dist worksteal` — зелёный, 26 новых тестов
  (`test_resume_pool.py`, `test_resume_pool_command.py`,
  `test_resume_cluster_config.py`), бюджет `check_pytest_budget.py` — в норме
  (24.4s / 240s).
- `scripts/gen_cli_docs.py --check` — README перегенерирован.
- Реальный `--dry-run` прогон против аккаунта из основного репозитория
  (резюме `marketing`, только для проверки плана — план не зависит от того,
  подходит ли резюме содержательно под IT-кластеры):

  ```
  $ hhru resume-pool --config <config.yaml> --source marketing --dry-run
  [DRY-RUN] Пул резюме от 'marketing' (resume_id 6b85a5a1ff05bc40630039ed1f6e346e537370): требуется создать 4 из 4 кластеров
    1. ai_llm         -> slug 'marketing-ai_llm' title-заготовка: AI/LLM-инженер
    2. data_science   -> slug 'marketing-data_science' title-заготовка: Data Science / ML
    3. data_engineer  -> slug 'marketing-data_engineer' title-заготовка: Data-инженер
    4. python_backend -> slug 'marketing-python_backend' title-заготовка: Python-бэкенд
  [INFO] Ничего не отправлено.
  ```

## Живой прогон подтверждён

Базовое резюме — существующее `marketing` (по решению пользователя;
содержательное соответствие кластерам не принципиально для #754, это работа
#769). Боевой `resume-pool --source marketing --force` создал все 4 копии
успешно (`attempted=4 success=4 failed=0 uncertain=0`), title каждой копии
совпал с title-заготовкой из dry-run плана, подтверждено через `list-resumes`
(все 4 видны на hh.ru в статусе "черновик"). Полный вывод — в комментариях
к этому PR.

Closes #754

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by axisrow** [comment]
### Code review

Пять параллельных ревью-агентов (CLAUDE.md compliance, shallow bug scan, git history/blame, прошлые PR-комментарии, соответствие inline-комментариям переиспользуемых функций) нашли 1 подтверждённый issue, исправлен в этом же PR (2ebfb7b).

1. **`_write_lock_path` не сериализовал `resume-pool` по account/config-директории**, хотя команда выполняет ту же account-wide `copy_resume_on_hh` list-diff reconciliation, что и `copy-resume`, несколько раз подряд за один запуск (по разу на кластер). Проверка была завязана буквально на имя команды:

https://github.com/axisrow/hhru/blob/b1555b0eb996e17e9dd48b76eb32be3f2f9c2ed/src/hhru_bot/cli.py#L293-L296

Это тот же race, что чинил PR #558 / коммит `a52306c` (два параллельных клона делят один `parentResumeId`, любой процесс может применить `--title` к чужому клону) — только `resume-pool` попадал под history-scoped lock вместо account-scoped, поэтому два параллельных запуска `resume-pool` (или `resume-pool` + `copy-resume`) с разными `--history` против одного аккаунта не сериализовались бы. Fix: `mutates_external_resume_list = args.command in ("copy-resume", "resume-pool")` + regression-тест `test_resume_pool_different_history_files_share_account_lock` (зеркалит существующий тест для `copy-resume`).

Остальные проверки (durable-гарантии `has_unresolved_uncertain`/`DurableMutationAttempt`, отсутствие прямого HTTP, корректность `--limit`, reuse `copy_resume.py`-хелперов, `_set_copy_title` внутри того же `launch_context`, отсутствие новой SQLite-схемы) — без нарушений. Одно minor-наблюдение без исправления: команда не печатает `[INFO] Уже копировали сегодня` (как `copy-resume`) — чисто UX-асимметрия, не инвариант.

🤖 Generated with [Claude Code](https://claude.ai/code)

<sub>- If this code review was useful, please react with 👍. Otherwise, react with 👎.</sub>

**Comment by axisrow** [comment]
## Живой прогон подтверждён

По решению пользователя базовым резюме использовано существующее `marketing`
(содержательное соответствие кластерам не принципиально для проверки #754 —
адаптация содержимого под кластер отдельный issue #769; цель этого прогона —
проверить механику команды).

### Боевой прогон (`--force`, без `--write-config`)

```
$ hhru --config <config.yaml> --headless resume-pool --source marketing --force
[OK] кластер ai_llm -> resume_id 66b35db0ff11063afc0039ed1f543435576930
[INFO] Добавьте в config.yaml:
         - id: "<придумайте-id>"
           resume_url: "https://hh.ru/resume/66b35db0ff11063afc0039ed1f543435576930"
[INFO] Добавьте вручную: cluster: "ai_llm"
[OK] кластер data_science -> resume_id 51b6b2bcff11063b330039ed1f6f6a74667936
[INFO] Добавьте в config.yaml:
         - id: "<придумайте-id>"
           resume_url: "https://hh.ru/resume/51b6b2bcff11063b330039ed1f6f6a74667936"
[INFO] Добавьте вручную: cluster: "data_science"
[OK] кластер data_engineer -> resume_id 3d292cf4ff11063b580039ed1f354756616935
[INFO] Добавьте в config.yaml:
         - id: "<придумайте-id>"
           resume_url: "https://hh.ru/resume/3d292cf4ff11063b580039ed1f354756616935"
[INFO] Добавьте вручную: cluster: "data_engineer"
[OK] кластер python_backend -> resume_id fb6ef122ff11063b840039ed1f3857524a7a30
[INFO] Добавьте в config.yaml:
         - id: "<придумайте-id>"
           resume_url: "https://hh.ru/resume/fb6ef122ff11063b840039ed1f3857524a7a30"
[INFO] Добавьте вручную: cluster: "python_backend"
[RUN] id=eb36c98d-763d-4fcd-94da-519bf94b9f94 status=completed attempted=4 success=4 failed=0 uncertain=0 skipped=0
```

Все 4 копии успешны с первой попытки — batch не останавливался, `uncertain=0`.

### Подтверждение на hh.ru (`list-resumes`)

```
| 66b35db0ff11063afc0039ed1f543435576930 | —  | AI/LLM-инженер     | черновик |
| 51b6b2bcff11063b330039ed1f6f6a74667936 | —  | Data Science / ML  | черновик |
| 3d292cf4ff11063b580039ed1f354756616935 | —  | Data-инженер       | черновик |
| fb6ef122ff11063b840039ed1f3857524a7a30 | —  | Python-бэкенд      | черновик |
```

`_set_copy_title` корректно установил title каждой копии под соответствующий
кластер (сравни с dry-run планом выше — точное совпадение title-заготовок).
Резюме в статусе "черновик" — ожидаемо для свежих копий, публикация вне scope
этого issue.

Резюме не удалены, `--write-config` не запускался (issue п.3: разрешение
только на создание) — связка `cluster` в config.yaml остаётся ручным шагом
пользователя (как и у `copy-resume`/`create-resume` без `--write-config`).

**Review by axisrow — COMMENTED** [review]
## Ревью PR #770 — resume-pool (#754)

**Вердикт: требуются изменения** (одна реальная находка, остальное — подтверждение корректности).

Реализация аккуратная и хорошо согласована с существующими инвариантами проекта. Проверил отдельно и подтверждаю как корректное:

- **Durable-гарантии не обходятся.** `has_unresolved_uncertain(source.resume_id, "copy_resume")` действительно ключуется по resume_id ИСХОДНОГО резюме, guard перепроверяется перед каждой итерацией (`resume_pool.py:164`), batch останавливается на первом `failed`/`uncertain`. `DurableMutationAttempt` используется по контракту: `before_click` резервирует intent только с момента реального клика, `interrupt()` при `action_id is None` — no-op, поэтому pre-click `NotAuthenticated` корректно уходит в `SESSION_EXPIRED` через `run_supervised_command`, а не в ложный `uncertain`.
- **Fail-closed контракты скопированы верно.** Проверка `new_resume_id` (совпал с исходным / пуст) и uncertain-контракт `_set_copy_title` совпадают с `copy-resume` (#116/#569); `_set_copy_title` вызывается внутри того же `with launch_context`, что и сама копия — та же поправка #569, не потеряна.
- **Write-lock.** Расширение `mutates_external_resume_list` на `resume-pool` (`cli.py:295`) обосновано: та же account-wide reconciliation, что у `copy-resume`, только в цикле. Тест `test_resume_pool_different_history_files_share_account_lock` это закрепляет.
- **Число кластеров не хардкодится**, `--limit` безопасно деградирует на 0/отрицательных (`max(limit, 0)` → пустой план + внятное `[INFO]`, без обращения к hh.ru — проверил).
- Регистрация команды покрыта во всех четырёх реестрах (`BROWSER_COMMANDS`, `WRITE_COMMANDS`, sandbox preflight, write-lock), маркеры тестов — по одному на файл, docs/cli-spec + README + CLAUDE.md согласованы с кодом.

### Что нужно поправить

Одна находка — см. инлайн-комментарий: коллизия slug проверяется только ПОСЛЕ необратимой мутации на hh.ru, тогда как `copy-resume` намеренно проверяет её до запуска браузера.

### Замечено, но не находка

`--write-config` копирует секцию `search` источника в каждую копию кластера дословно (`replace(source, cluster=...)` → `_resume_mapping`). Это ровно то же поведение, что у `copy-resume --write-config`, и адаптация содержимого — работа #769, так что в рамках #754 это корректно. Упоминаю только чтобы зафиксировать: следующий `apply` без `--resume` пойдёт по всем четырём копиям с одинаковым поисковым запросом источника.

File: src/hhru_bot/commands/resume_pool.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+            print(f"[OK] кластер {item.cluster.key} -> resume_id {result.new_resume_id}")
+            if write_config:
+                try:
+                    write_resume_config(
+                        args.config,
+                        replace(source, cluster=item.cluster.key),
+                        item.slug,
+                        result.new_resume_id,
+                    )
+                except Exception as exc:  # noqa: BLE001 - копия уже необратима на hh.ru
```
Comment (axisrow) [inline src/hhru_bot/commands/resume_pool.py:252]: **Коллизия slug проверяется только после необратимой мутации на hh.ru.**

`copy-resume` намеренно валидирует slug ДО запуска браузера (`commands/copy_resume.py:382-393`), с явным обоснованием в комментарии: «Проверки записи — ДО браузера: отказ после успешного копирования на hh.ru оставил бы лишний дубль в аккаунте (та же логика, что у проверки коллизии слагов выше — отказ до записи, #520)». Здесь этой предварительной проверки нет вовсе — единственная защита это `write_resume_config`, которая срабатывает уже после клика «Дублировать».

**Сценарий отказа.** В `config.yaml` есть резюме `id: "backend-ai_llm"` без поля `cluster:` — например, копия, созданная предыдущим прогоном `resume-pool` БЕЗ `--write-config` и добавленная пользователем вручную по напечатанному YAML-фрагменту (команда сама печатает `[INFO] Добавьте вручную: cluster: "..."` отдельной строкой, так что забыть дописать `cluster` — штатный путь, а не экзотика).

Проверил на текущем коде: `build_pool_plan` считает кластер непокрытым (он смотрит только `r.cluster`, не `r.id`) и выдаёт `PoolPlanItem(slug="backend-ai_llm")`. Дальше цикл делает реальный клик «Дублировать», копия на hh.ru создаётся, и только потом `write_resume_config` → `load_config(candidate)` падает на `ConfigError("Дублирующийся id резюме в конфиге: backend-ai_llm")` (`config.py:237`). Итог: `[FAIL] Копия создана, но config.yaml не обновлён`, `any_failed = True`, `break` — в аккаунте остаётся лишний дубль резюме, который нельзя откатить, а весь оставшийся пул не создан.

**Предлагаемая правка:** проверять коллизии slug для всех `plan.items` разом до `launch_context` — рядом с уже существующим `reject_bare_source(source)` (строка 120), пока ни одной мутации не произошло:

```python
if write_config:
    existing = {item.id for item in config.resumes} | {item.resume_id for item in config.resumes}
    clashing = [i.slug for i in plan.items if i.slug in existing]
    if clashing:
        print(f"[FAIL] Slug уже есть в конфиге: {'", "'.join(clashing)}")
        sys.exit(1)
```

Это ровно тот же fail-closed принцип «отказ до записи», что уже зафиксирован в `copy-resume`, и он тем важнее для batch-режима: цена ошибки здесь — до N необратимых дублей вместо одного.

**Comment by axisrow** [comment]
Обе находки исправлены в 8d43665:

1. **Коллизия slug (blocking)** — теперь весь план (`plan.items`) проверяется против существующих `id`/`resume_id` конфига ДО `launch_context`, тем же принципом «отказ до записи» (#520), что у `copy-resume` (`commands/copy_resume.py:386-389`). При коллизии — `[FAIL]` и `sys.exit(1)` без единого клика на hh.ru.
2. **`reject_bare_source` безусловно** (найдено параллельно `/review`) — теперь гейтится `write_config`, как у `copy-resume` (`commands/copy_resume.py:392-393`): проверка нужна только для секции, которую пишет `--write-config`.

Добавлены 2 регрессионных теста: `test_run_refuses_slug_collision_before_touching_browser`, `test_run_allows_bare_source_without_write_config`. Полный `pytest` (2864 passed), `ruff check` + `ruff format --check` зелёные.

**Review by axisrow — COMMENTED** [review]
## Повторное ревью PR #770 — находка закрыта

**Вердикт: готово к мержу.** Блокирующая находка исправлена в `8d43665`, проверил на реальном коде.

### Исправление проверено

`src/hhru_bot/commands/resume_pool.py:143-156` — проверка коллизии slug теперь стоит там, где нужно: до `confirm_write`, до `History`, до любого `launch_context`. Проверяется весь `plan.items` разом, а не поэлементно, поэтому batch останавливается ещё до первого клика «Дублировать».

**Тест — настоящий регрессионный страж, а не тавтология.** Проверил прямо: подменил `resume_pool.py` версией из `2ebfb7b` и прогнал `test_run_refuses_slug_collision_before_touching_browser` — тест падает, и падает ровно тем сценарием, который я описал в находке:

```
[FAIL] Копия создана, но config.yaml не обновлён: ...
[RUN] status=partial attempted=1 success=1 failed=0 uncertain=0
```

То есть на старом коде копия на hh.ru действительно создавалась (`success=1`) и лишь потом ломалась запись конфига. На новом коде тест зелёный и `env.calls == []` — до браузера дело не доходит.

### Про вторую правку в коммите (`reject_bare_source` под `write_config`)

Это изменение я не просил, поэтому проверил отдельно — **оно верное, и это моя недосмотренная находка, а не расширение объёма.**

`copy-resume` гейтит `reject_bare_source` ровно так же — внутри `if write_config:` (`commands/copy_resume.py:382-393`), вместе с проверкой слагов. В `resume-pool` вызов стоял безусловно, из-за чего `--dry-run` по bare-резюме (#319) падал `[FAIL]`, хотя dry-run вообще ничего не пишет. Подтвердил на `bare_resume(...)`: `search.text` пустой, но без `--write-config` он ни на что не влияет — секция в `config.yaml` не создаётся, а именно её и защищает `reject_bare_source`. Так что гейт восстанавливает паритет с `copy-resume`, а не ослабляет инвариант.

Отличие от `copy-resume`, которое здесь допустимо: там дополнительно отбраковываются пустые слаги и слаги с переводом строки. В `resume-pool` slug генерируется машинно (`{source.id}-{cluster.key}`), пользовательского ввода в нём нет, поэтому проверка не нужна.

### Прогон

- `pytest -n 4 --dist worksteal` — вся сюита зелёная.
- `ruff check` + `ruff format --check` по `src tests scripts` — чисто.
- Целевые файлы: 28 тестов зелёные.

Остальные инварианты, подтверждённые в первом ревью (durable-гарантии, контракты #116/#569, write-lock, `--limit`, регистрация в реестрах, документация), коммитом не затронуты.
